<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLoad</class>
 <widget class="QMainWindow" name="frmLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create load</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxPort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Port1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Port2</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Comlex impendence(Ohm)</string>
    </property>
    <widget class="QLabel" name="lblReal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Real part</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblImag">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Imaginary part</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtReal">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtImag">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtReceive">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblReal_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive(degree)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLoadName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoadName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
